test/check-away-testing.sh

```diff
@@ -0,0 +1,59 @@
+FAILED=0
+TOTAL=0
+SKIPPED=0
+
+EXCLUDE=("stations" "geminus_city")
```

Comment:
i pre-empted you on this a bit, you can remove geminus from this as the folder no longer exists

```suggestion
EXCLUDE=("stations")
```